modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/beefman.dm

```diff
@@ -0,0 +1,519 @@
+			if (istype(I, /obj/item/food/meat/slab))
+				user.put_in_hands(I)
+
+			return TRUE
+	return ..()
+
+/datum/species/beefman/spec_unarmedattacked(mob/living/carbon/human/user, mob/living/carbon/human/target)
+	// Bleed On
+	if (user != target && user.is_bleeding())
+		target.add_mob_blood(user) //  from atoms.dm, this is how you bloody something!s
+	return ..()
+
+/datum/species/beefman/proc/handle_limb_mashing()
+	SIGNAL_HANDLER
+
+/datum/species/beefman/spec_attacked_by(obj/item/I, mob/living/user, obj/item/bodypart/affecting, mob/living/carbon/human/H, modifiers)
+	handle_limb_mashing()
+	// MEAT LIMBS: If our limb is missing, and we're using meat, stick it in!
+	if(LAZYACCESS(modifiers, RIGHT_CLICK))
+		return
+	if(H.stat < DEAD && !affecting && istype(I, /obj/item/food/meat/slab))
+		var/target_zone = user.zone_selected
+		var/list/limbs = list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+
+		if((target_zone in limbs))
+			if(user == H)
+				user.visible_message("[user] begins mashing [I] into [H]'s torso.", span_notice("You begin mashing [I] into your torso."))
+			else
+				user.visible_message("[user] begins mashing [I] into [H]'s torso.", span_notice("You begin mashing [I] into [H]'s torso."))
+
+			// Leave Melee Chain (so deleting the meat doesn't throw an error) <--- aka, deleting the meat that called this very proc.
+			spawn(1)
+				if(do_mob(user,H))
+					// Attach the part!
+					var/obj/item/bodypart/newBP = H.newBodyPart(target_zone, FALSE)
+					H.visible_message("The meat sprouts digits and becomes [H]'s new [newBP.name]!", span_notice("The meat sprouts digits and becomes your new [newBP.name]!"))
+					newBP.attach_limb(H)
+					if(I.reagents && I.reagents.total_volume)
+						I.reagents.trans_to(user, I.reagents.total_volume)	//transfer reagents to player
+					qdel(I)
+					playsound(get_turf(H), 'sound/effects/meatslap.ogg', 50, 1)
+
+			return TRUE // True CANCELS the sequence.
+
+	return ..() // TRUE FALSE
+
+/datum/species/beefman/before_equip_job(datum/job/playerJob, mob/living/carbon/human/H)
+
+	// Pre-Equip: Give us a sash so we don't end up with a Uniform!
+	var/obj/item/clothing/under/bodysash/newSash
+	switch(playerJob.title)
+
+		// Assistant
+		if("Assistant")
+			newSash = new /obj/item/clothing/under/bodysash()
+		// Captain
+		if("Captain")
+			newSash = new /obj/item/clothing/under/bodysash/captain()
+		// Security
+		if("Head of Security")
+			newSash = new /obj/item/clothing/under/bodysash/hos()
+		if("Warden")
+			newSash = new /obj/item/clothing/under/bodysash/warden()
+		if("Security Officer")
+			newSash = new /obj/item/clothing/under/bodysash/security()
+		if("Detective")
+			newSash = new /obj/item/clothing/under/bodysash/detective()
+		if("Brig Physician")
+			newSash = new /obj/item/clothing/under/bodysash/brigdoc()
+		if("Deputy")
+			newSash = new /obj/item/clothing/under/bodysash/deputy()
```

Comment:
are you sure you have these jobs in the first place?

Reply:
we have the former but not the latter